src/lcmtype.jl

```diff
@@ -168,60 +182,55 @@ end
     exprs = Expr[]
```

Comment:
I'm no longer using `resize!` in `decodefield!`, but I think it may still be useful. Needs a separate test now though.

Reply:
Done.